falsepositive_regex.list

```diff
@@ -3,3 +3,4 @@ campaign-statistics.com
 nvidia.com
 rtu.lv
 vuagame.store
+pichinchamasbeneficios.pe/PichinchaPeWeb/login
```

Comment:
You should use https://github.com/mitchellkrogza/phishing/blob/main/add-link as this is a domain only list (https://github.com/mitchellkrogza/phishing?tab=readme-ov-file#false-positives)

I agree the README could use a little hand to make this more clear